source/Plugins/Language/Swift/SwiftFormatters.cpp

```diff
@@ -322,7 +369,8 @@ bool lldb_private::formatters::swift::StringGuts_SummaryProvider(
       // Small strings aren't emitted yet.
       return false;
     } else {
-      // The associated value in the selected enum case is an NSString.
+      // The associated value in the selected enum case is either an NSString,
```

Comment:
Does the variant case not tell you which of these two possibilities was chosen?  It bugs me that we have to guess.